### ISSUE TITLE
copy/paste in geo image 

### ISSUE THREAD

**Issue by chritter** [StatCan/kubeflow-containers-desktop#23]
Hello @frazs ,

It would be great if the image paste functionality would be enabled for the geomatics image. This way we can paste images/screenshots from our geo analysis in the notebooks. Would this be possible? Below is an example of an add-on in jupyter lab which supports that.

![image](https://user-images.githubusercontent.com/15911039/86492211-5e3d0980-bd3b-11ea-8bdd-bed0797eb27f.png)

**Comment by frazs** [comment]
Make a cell where you want to paste one or more images. Select it, and in your top toolbar, where it says "Code":
![image](https://user-images.githubusercontent.com/13685849/86970032-b78ea980-c13c-11ea-8524-7bc5e9c46109.png)

Click on "Code" and change it to "Markdown". Try to paste your image. I can do this with `Ctrl+v`. I can also do this by holding `Shift` while right-clicking the cell and then selecting `Paste`.

You should see a string appear, such as for example `![image.png](attachment:cb202a38-6fa6-4ee9-85ba-4e7f5431c0f1.png)`. 

If you run that cell, while still having Markdown selected, those image strings should turn into images. As far as I can tell, pasting images directly in a `Code` block, in between code that will be evaluated, is not supported.

The `jupyter-clipboard` extension is deprecated because this functionality has been merged into the version of jupyterlab we are using.

Let me know if you find a paste method that works. If you do not, this may be another Mac support issue.

**Comment by frazs** [comment]
Dragging an image into a Markdown cell can also work

**Comment by frazs** [comment]
Review blocked by https://github.com/StatCan/kubeflow-containers/issues/77

**Comment by chritter** [comment]
@frazs Yes, your suggestion works! Thank you. I would close this issue.
